### PR TITLE
[RC2] Restructuring chapter structure and adding reading guide

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 [<< Back](../)
 
-# 2. Kubernetes Conformance Requirements
+# 3. Kubernetes Test Cases and Traceability to CNTT Requirements
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/chapters/chapter02.md:3]: I think there is a drive to not use the term "CNTT" - suggestion below
```suggestion
# 3. Kubernetes Test Cases and Requirements Traceability
```

File: doc/ref_cert/RC2/README.md [hunk 2]
```diff
@@ -24,20 +24,16 @@ This is the Kubernetes Based Reference Conformance (RC-2)
...
 | --- | --- |
 | Chapter 01 - Introduction                                                 | Initial Framework Only |
-| Chapter 02 - Kubernetes Conformance Requirements                          | Initial Framework Only |
-| Chapter 03 - Kubernetes Test Cases and Traceability to CNTT Requirements  | Initial Framework Only |
-| Chapter 04 - Kubernetes Testing Cookbook                                  | Initial Framework Only |
-| Chapter 05 - CNF Conformance Requirements                                 | Initial Framework Only |
-| Chapter 06 - CNF Test Cases and Traceability to CNTT Requirements         | Initial Framework Only |
-| Chapter 07 - CNF Testing Cookbook                                         | Initial Framework Only |
-| Chapter 08 - Gap analysis and Development                                 | Initial Framework Only |
+| Chapter 02 - Kubernetes Test Cases and Traceability to CNTT Requirements  | Initial Framework Only |
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/README.md:27]: I think there is a drive to not use the term "CNTT" - suggestion below

```suggestion
| Chapter 02 - Kubernetes Test Cases and Requirements Traceability | Initial Framework Only |
```

File: doc/ref_cert/RC2/README.md [hunk 3]
```diff
@@ -24,20 +24,16 @@ This is the Kubernetes Based Reference Conformance (RC-2)
...
+| Chapter 03 - Kubernetes Testing Cookbook                                  | Initial Framework Only |
+| Chapter 04 - CNF Test Cases and Traceability to CNTT Requirements         | Initial Framework Only |
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/README.md:29]: I think there is a drive to not use the term "CNTT" - suggestion below

```suggestion
| Chapter 04 - CNF Test Cases and Requirements Traceability | Initial Framework Only |
```

File: doc/ref_cert/RC2/README.md [hunk 4]
```diff
@@ -24,20 +24,16 @@ This is the Kubernetes Based Reference Conformance (RC-2)
...
 ## Table of Contents
 * [Chapter 01 - Introduction](chapters/chapter01.md)
-* [Chapter 02 - Kubernetes Conformance Requirements](chapters/chapter02.md)
-* [Chapter 03 - Kubernetes Test Cases and Traceability to CNTT Requirements ](chapters/chapter03.md)
-* [Chapter 04 - Kubernetes Testing Cookbook](chapters/chapter04.md)
-* [Chapter 05 - CNF Conformance Requirements](chapters/chapter05.md)
-* [Chapter 06 - CNF Test Cases and Traceability to CNTT Requirements](chapters/chapter06.md)
-* [Chapter 07 - CNF Testing Cookbook](chapters/chapter07.md)
-* [Chapter 08 - Gap analysis and Development](chapters/chapter08.md)
+* [Chapter 02 - Kubernetes Test Cases and Traceability to CNTT Requirements ](chapters/chapter02.md)
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/README.md:35]: I think there is a drive to not use the term "CNTT" - suggestion below

```suggestion
* [Chapter 02 - Kubernetes Test Cases and Requirements Traceability](chapters/chapter02.md)
```

File: doc/ref_cert/RC2/README.md [hunk 5]
```diff
@@ -24,20 +24,16 @@ This is the Kubernetes Based Reference Conformance (RC-2)
...
+* [Chapter 03 - Kubernetes Testing Cookbook](chapters/chapter03.md)
+* [Chapter 04 - CNF Test Cases and Traceability to CNTT Requirements](chapters/chapter04.md)
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/README.md:37]: I think there is a drive to not use the term "CNTT" - suggestion below

```suggestion
* [Chapter 04 - CNF Test Cases and Requirements Traceability](chapters/chapter04.md)
```

File: doc/ref_cert/RC2/chapters/chapter04.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 [<< Back](../)
 
-# 4. Kubernetes Testing Cookbook
+# 4. CNF Test Cases and Traceability to CNTT Requirements
```
Comment (tomkivlin) [inline doc/ref_cert/RC2/chapters/chapter04.md:3]: I think there is a drive to not use the term "CNTT" - suggestion below

```suggestion
# 4. CNF Test Cases and Requirements Traceability
```

**Review by tomkivlin — CHANGES_REQUESTED** [review]

**Comment by xmulligan** [comment]
@tomkivlin thanks, I've accepted your changes to remove CNTT

**Review by tomkivlin — APPROVED** [review]

**Comment by scottsteinbrueck** [comment]
@xmulligan @tomkivlin Can you work to resolve conflicts and prepare for PR merge ?  Thanks.

**Comment by collivier** [comment]
Removing empty chapters should rather occur at the beginning not at the end of the development period.
The merge looks good but I do double check it.

It's mostly unreviewable due to the huge amount of commits. Only 1.6 is now right?

**Review by collivier — CHANGES_REQUESTED** [review]
Would you mind formating tabs?

**Comment by xmulligan** [comment]
@scottsteinbrueck the conflicts have been removed
@collivier I formatted the tabs

Can you both check again?

**Comment by scottsteinbrueck** [comment]
@collivier if you review / approve, then I will add my approve / merge.
@xmulligan FYI

**Comment by collivier** [comment]
@scottsteinbrueck we shouldn't merge restructuring during proof reading. As @tomkivlin managed the most difficult part, let's check if it's safe enough

**Comment by collivier** [comment]
It would be great to manage git in the appropriate way.
Pushing so many commits is good for stats but not for rebasing branches.

**Comment by collivier** [comment]
@xmulligan your last proposal simply remove contents which are important. Am I allowed to fix all history here?

**Review by collivier — APPROVED** [review]
Let's merge it as it seems now clean

**Comment by xmulligan** [comment]
@collivier thanks for help changing the PR and getting it merged

**Pull request merged**